recipes-bsp/acpi-tables/samples/edison/sc16is7xx.asl

```diff
@@ -0,0 +1,69 @@
+/*
+ * Intel Edison
+ *
+ * This adds NXP SC16IS7XX I2C-UART converter to the
+ * I2C#6  bus available on Intel Edison:
+ *
+ *   pin name           pin number
+ *   -----------------------------
+ *   I2C_6_SDA       112
+ *   I2C_6_SCL       111
+ *
+ * In Linux you need to set CONFIG_SERIAL_SC16IS7XX_CORE=y (or m) to be able to use
+ * this device.
+ *
+ * Copyright (C) 2018, Emlid Limited
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+DefinitionBlock ("serial.aml", "SSDT", 5, "", "SERIAL", 1)
+{
+    External (_SB_.PCI0.I2C6, DeviceObj)
+
+    Scope (\_SB.PCI0.I2C6)
+    {
+        Device (SER)
+        {
+            Name (_HID, "PRP0001")
+            Name (_DDN, "SC16IS7XX I2C-SERIAL")
+            Name (_CRS, ResourceTemplate () {
+                I2cSerialBus (
+                    0x4d,                   // Bus address
+                    ControllerInitiated,    // Don't care
+                    100000,                 // 100 kHz
```

Comment:
Can you remind me, what we decide to do with the speed? I think this one is not enough to support 115200.

Reply:
I believe you suggested 400kHz. I think we were waiting for @staroselskii to test this?

Reply:
400Khz sounds right. We've been using 57600 and 9600 for quite some time with 100Khz on I2C w/o any issues. 400Khz won't do any harm but we'll test today and report back.
@kobylyanskiy

Reply:
@andy-shev @htot I've tested both tables compiled with 100Khz and 400Khz on 115200, everything was fine. I didn't see any issues with testing it under load. Looks good to me.